frontend/test/metabase/scenarios/question/notebook.cy.spec.js

```diff
@@ -21,418 +21,444 @@ const { ORDERS, ORDERS_ID, PEOPLE, PEOPLE_ID, PRODUCTS, PRODUCTS_ID } =
     });
 
-    cy.createQuestion({
-      name: "Orders model",
-      query: { "source-table": ORDERS_ID },
-      dataset: true,
-      display: "table",
+    it("should prompt to join with a model if the question is based on a model", () => {
+      cy.createQuestion({
+        name: "Products model",
+        query: { "source-table": PRODUCTS_ID },
+        dataset: true,
+        display: "table",
+      });
+
+      cy.createQuestion({
+        name: "Orders model",
+        query: { "source-table": ORDERS_ID },
+        dataset: true,
+        display: "table",
+      });
+
+      startNewQuestion();
+      popover().findByText("Models").click();
+      popover().findByText("Products model").click();
+      join();
+      popover().findByText("Orders model").click();
+      popover().findByText("ID").click();
+      popover().findByText("Product ID").click();
+
+      visualize();
     });
 
-    startNewQuestion();
-    popover().findByText("Models").click();
-    popover().findByText("Products model").click();
-    join();
-    popover().findByText("Orders model").click();
-    popover().findByText("ID").click();
-    popover().findByText("Product ID").click();
+    // flaky test
+    it.skip("should show an info popover when hovering over a field picker option for a table", () => {
+      startNewQuestion();
+      cy.contains("Sample Database").click();
+      cy.contains("Orders").click();
 
-    visualize();
-  });
+      cy.findByTestId("fields-picker").click();
 
-  // flaky test
-  it.skip("should show an info popover when hovering over a field picker option for a table", () => {
-    startNewQuestion();
-    cy.contains("Sample Database").click();
-    cy.contains("Orders").click();
+      cy.findByText("Total").trigger("mouseenter");
 
-    cy.findByTestId("fields-picker").click();
+      popover().contains("The total billed amount.");
+      popover().contains("80.36");
+    });
 
-    cy.findByText("Total").trigger("mouseenter");
+    // flaky test
+    it.skip("should show an info popover when hovering over a field picker option for a saved question", () => {
+      cy.createNativeQuestion({
+        name: "question a",
+        native: { query: "select 'foo' as a_column" },
+      });
 
-    popover().contains("The total billed amount.");
-    popover().contains("80.36");
+      // start a custom question with question a
+      startNewQuestion();
+      cy.findByText("Saved Questions").click();
+      cy.findByText("question a").click();
+
+      cy.findByTestId("fields-picker").click();
+
+      cy.findByText("A_COLUMN").trigger("mouseenter");
+
+      popover().contains("A_COLUMN");
+      popover().contains("No description");
+    });
   });
 
-  // flaky test
-  it.skip("should show an info popover when hovering over a field picker option for a saved question", () => {
-    cy.createNativeQuestion({
-      name: "question a",
-      native: { query: "select 'foo' as a_column" },
+  describe("postgres", () => {
```

Comment:
You would also need to add an `external` tag so this test is excluded locally if not postgres instance is available